var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -41,6 +41,7 @@ class Acts(CMakePackage, CudaPackage):
     # Supported Acts versions
     version("main", branch="main")
     version("master", branch="main", deprecated=True)  # For compatibility
+    version("38.1.0", commit="8a20c88808f10bf4fcdfd7c6e077f23614c3ab90", submodules=True)
```

Comment:
https://github.com/acts-project/acts/pull/3921 implies that acts@:38.0 is not supporting edm4hep@0.99: so we may need to add a conflict (probably easier than adding this through version ranges).

do you have any intentation to keep track of compiler compatibility, e.g. https://github.com/acts-project/acts/pull/3938?

Reply:
Updated by adding a conflict as you suggested.